### ISSUE TITLE
add author to posts, connect it to github user

### ISSUE THREAD

**Issue by okvv** [Negev-Web-Dev/Negev-Web-Dev.github.io#4]
research if possible in current stack

**Comment by yanai101** [comment]
I can help with that -  we can get the medium post by username - and display it  :)

**Comment by okvv** [comment]
no, the entire post is a commit to repo so I want the user who did the commit to be the author (and show it on the post)
so it's two task,
BE: figure out how to associate github user to post
FE: display github profile on post as author
